src/link.ts

```diff
@@ -486,7 +490,7 @@ export class LinkContainer {
     idTypeFilter : <(idtype: IDType, i: number, dataVis: IDataVis) => boolean>constantTrue,
```

Comment:
see comment before, where I created the function again to avoid writing it 3-4 times